cafebabel/templates/500.html

```diff
@@ -0,0 +1,31 @@
+{# Beware of not adding any external dependencies here! #}
```

Comment:
I would also consider avoiding `url_for` and `include` in this file.
That would mean hardcoding the paths and images url tho.

Reply:
yep: https://github.com/cafebabel/cafebabel.com/pull/282/commits/97b39cdd8ad8fcc1fa348738be2db026fdf6c882